region/norfair/east/Cathedral Entrance.json

```diff
@@ -155,6 +167,44 @@
         {"heatFrames": 150}
       ]
     },
+    {
+      "link": [1, 5],
+      "name": "Cathedral Entrance 10 Power Bomb Crystal Flash",
+      "notable": true,
+      "reusableRoomwideNotable": "Cathedral Entrance 10 Power Bomb Crystal Flash",
+      "entranceCondition": {
+        "comeInRunning": {
+          "minTiles": 0.0625,
+          "maxTiles": 0.0625,
+          "speedBooster": false
+        }
+      },
+      "requires": [
+        "canPrepareForNextRoom",
+        {"heatFrames": 540},
+        "h_can10PowerBombCrystalFlash"
+      ],
+      "note": [
+        "Normalized movement through the room can be used to manipulate the Sm. Dessgeegas and do this strat reliably without heat protection.",
+        "The following is one way to do this; other approaches may also be possible.",
+        "Before entering the room, damage down to between 127 and 159 energy, and unequip Speed Booster if it is collected.",
+        "Position one pixel away from the door in the previous room.",
+        "Hold angle-down (and optionally dash), enter the door transition, then hold shot while continuing to hold forward, angle-down, and dash.",
+        "The shot block will be destroyed by the buffered shot.",
+        "Continue holding right and dash to run under both Sm. Dessgeegas.",
+        "After running under the second one, stop for a moment to prevent it from scrolling off camera.",
+        "Morph, roll all the way against the right wall and quickly lay a Power Bomb.",
+        "Move slightly to the left to avoid being bounced by the Power Bomb explosion.",
+        "As the Sm. Dessgeega approaches, roll under it to the left to avoid being hit.",
```

Comment:
I'm seeing about a second to quickly get on the right pixel, and I'd rather have 2.
And quickly killing the left hopper without stopping makes the room entry less strict while the right hopper still jumps over samus.

Reply:
Makes sense, increased the heat frames by another 30, bringing the total leniency to at least full second.

Just tried playing around some more with entering from other pixels. There is a large range where you can kill the first hopper and run under the second, but then the behavior of the second afterward seems chaotic. I guess it depends on the exact position where you stop and for how long, to prevent it going off-camera. It works sometimes, but other times it doesn't end up in a place where you can collect the Power Bomb and return in time. Same thing with letting it go off-camera and then bringing it back on. So far the last-pixel entrance is the only one for me that seems to make it reliable where it will move in the correct way 100% of the time. Though maybe with a better understanding of the hopper AI the other entrances could work, if there is some way to manipulate it to consistently do the right kind of jumps.